functions.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * @param PDO $db the db conn
+ * @return string
+ */
+
+$servername = "localhost";
+$username = "root";
+$password = "password";
+$db = "vinyl-collection";
+
+function retrieveDb(): PDO
+{
+    $db = new PDO('mysql:host=db; dbname=vinyl-collection', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+$db = retrieveDb();
+
+$query = "SELECT `image`, `artist`, `album`, `year`, `genre`, `label-released` FROM `vinyl-collection`;";
+
+
+$stmnt = $db->prepare($query);
+
+$stmnt->execute();
+
+$result = $stmnt->fetchAll ();
+
+echo '<pre>';
```

Comment:
Once you've functionalised the select statement as per Stephs comment, this should be move to index.php

Reply:
done